src/main/resources/css/cucumber.css

```diff
@@ -210,6 +210,14 @@ tr:hover {
     color: #eee;
 }
 
+.white-space-normal {
+    white-space: normal;
```

Comment:
Default behave overriden by browser's stylesheets in Chrome and Firefox. Maybe in others browsers too.